pkg/xds/envoy/clusters/configurers.go

```diff
@@ -143,36 +129,15 @@ func PassThroughCluster(name string) ClusterBuilderOpt {
 	})
 }
 
-func StaticCluster(name string, address string, port uint32) ClusterBuilderOpt {
-	return ClusterBuilderOptFunc(func(config *ClusterBuilderConfig) {
-		config.AddV3(&v3.StaticClusterConfigurer{
-			Name:           name,
-			LoadAssignment: endpoints_v3.CreateStaticEndpoint(name, address, port),
-		})
-		config.AddV3(&v3.AltStatNameConfigurer{})
-		config.AddV3(&v3.TimeoutConfigurer{})
-	})
-}
-
-func StaticClusterUnixSocket(name string, path string) ClusterBuilderOpt {
-	return ClusterBuilderOptFunc(func(config *ClusterBuilderConfig) {
-		config.AddV3(&v3.StaticClusterConfigurer{
-			Name:           name,
-			LoadAssignment: endpoints_v3.CreateStaticEndpointUnixSocket(name, path),
-		})
-		config.AddV3(&v3.AltStatNameConfigurer{})
-		config.AddV3(&v3.TimeoutConfigurer{})
-	})
-}
-
-func StrictDNSCluster(name string, endpoints []core_xds.Endpoint, hasIPv6 bool) ClusterBuilderOpt {
+func EndpointCluster(name string, hasIPv6 bool, endpoints ...core_xds.Endpoint) ClusterBuilderOpt {
```

Comment:
The alternative is `EdsCluster` that also has "endpoint" in the name. I think to make it the opposite to `EdsCluster` it should be named `StaticEndpointCluster` or something like that

Reply:
Hmm, but at the same time StaticDNS cluster doesn't really have "static endpoints". Maybe a simple `StaticCluster` name is better

Reply:
I was thinking of something with Static but yes I don't want to make it confusing

Reply:
`ProvidedEndpointCluster` maybe? As the hosts are provided already.